ansys/api/additive/v0/additive_simulation.proto

```diff
@@ -72,6 +74,9 @@ message DownloadFileRequest {
 	string remote_file_name = 1; // full path to file on server
 }
 
+message DownloadLogsRequest {
```

Comment:
Since it's empty, I'd prefer we not add this message unless we have to for syntax reasons.